src/App.jsx

```diff
@@ -14,45 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'Sort alphabetically';
+const SORT_BY_LENGTH = 'Sort by length';
+
+function getSortedGoods(goods, sortField, reversed) {
+  let sortedGoods = [...goods];
+
+  if (sortField) {
+    switch (sortField) {
+      case SORT_ALPHABETICALLY:
+        sortedGoods = sortedGoods.sort((a, b) => a.localeCompare(b));
+        break;
+
+      case SORT_BY_LENGTH:
+        sortedGoods = sortedGoods.sort((a, b) => a.length - b.length);
+        break;
+
+      default:
+        return 0;
```

Comment:
method sort is duplicated, it's better to use one method sort and set condition inside the sort method, for instance:

```
sortedGoods.sort((a,b) => {
 switch (sortField) {
     case SORT_ALPHABETICALLY:
       return a.localeCompare(b)
       break;
})
```